### ISSUE TITLE
Feature Request: XSLT version 1 stylesheets

### ISSUE THREAD

**Issue by jscancella** [filak/hOCR-to-ALTO#24]
I have been having trouble getting XSLT version 2 stylesheets (like hocr__alto2.0.xsl) to work in python(This is partly in reference to #21). I would it be possible to create equivalent XSLT version 1 stylesheets for the hocr to alto?

**Comment by filak** [comment]
Sorry for the late reply.

The transformations have been intentionally implemented with XSLT 2.0 - otherwise it would be very cumbersome to do in XSLT 1.0 (though not impossible) But I have no plans nor time to do it with XSLT 1.0 

I am aware of the difficulties with Python XSLT 2.0 support - the last time I have checked it had not been very satisfying.

When I need to transform a xml file with XSLT 2.0 stylesheet using Python I just use subprocess call with SaxonHE - ie. (sample code simplified):

````
import subprocess

def transform(sourcedir, targetdir, style):

    xsl_processor_call = 'java -jar resources/saxon/saxon-'+SAXON_VERSION+'.jar'
    pargs = xsl_processor_call.split()
    pargs.append('-s:'+sourcedir)
    pargs.append('-xsl:'+style)
    pargs.append('-o:'+targetdir)  

    try:
        subprocess.CREATE_NO_WINDOW
        res_info = subprocess.check_output(pargs)

    except subprocess.CalledProcessError as err:
        print('ERROR: Invoking Saxon')

    except Exception as err:
        print('ERROR: Transformation')

    return res_info
````

**Comment by filak** [comment]
Late update: I have just recently found out there is an official SaxonHE Python lib: https://pypi.org/project/saxonche/
